src/client/js/partials/add-email.js

```diff
@@ -41,8 +41,8 @@ async function handleSubmit (e) {
   } catch (e) {
     // TODO: localize error messages
```

Comment:
Just wondering when we'd implement this TODO - is there a ticket for it?

Reply:
Yes – ticket is linked in description and was discussed yesterday 👍

Reply:
Dammit - PR descriptions get removed from my short term memory the moment I open the Changes tab :sweat_smile: